src/app/service/util.service.ts

```diff
@@ -8,4 +7,7 @@ export class UtilService {
 
   constructor() { }
 
+  hideLoader(): void {
+    document.getElementById("loader")!.style.display="none";
```

Comment:
use a subject instead... also kavil has added the same thing?

Reply:
Yeah